councilmatic/settings.py

```diff
@@ -185,11 +210,23 @@
 GOOGLE_API_KEY = env("GOOGLE_API_KEY")
 
 # - AWS
-AWS_KEY = env("AWS_KEY")
-AWS_SECRET = env("AWS_SECRET")
+AWS_S3_ACCESS_KEY_ID = env("AWS_S3_ACCESS_KEY_ID")
+AWS_S3_SECRET_ACCESS_KEY = env("AWS_S3_SECRET_ACCESS_KEY")
+AWS_STORAGE_BUCKET_NAME = env("AWS_STORAGE_BUCKET_NAME")
```

Comment:
Django Storages automatically discovers these variables.

Reply:
neat! that makes things easier. if we're removing these non-S3 specific vars here, what do you think of these heroku config var adjustments as well?

**staging** - in addition to `AWS_KEY` and `AWS_SECRET`, there's also `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` which seem to hold duplicate values of their S3 counterparts. i imagine these can all be removed

**production** - the S3 versions do not exist, so we can likely change `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to match the S3 var names. django storages also finds these automatically, but i'm unsure if the mismatch would cause issues considering these `settings` are looking for the S3 specific versions.

review apps are using only the S3 specific variables, so no change there